package.json

```diff
@@ -10,13 +10,16 @@
   },
   "dependencies": {
     "@next/font": "13.2.3",
-    "@stytch/nextjs": "^9.0.0",
-    "@stytch/vanilla-js": "^2.0.1",
+    "@stytch/nextjs": "^12.0.0",
+    "@stytch/vanilla-js": "^3.0.1",
     "eslint": "8.33.0",
     "eslint-config-next": "13.2.3",
-    "next": "13.1.6",
+    "next": "13.5.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "stytch": "^8.2.0"
+    "stytch": "^9.0.1"
+  },
+  "devDependencies": {
+    "@types/react": "18.2.28"
```

Comment:
no types for anything else?

Reply:
I believe this was an artifact from previous testing in this repo; this doesn't contain TypeScript officially.